test/appium/tests/atomic/chats/test_one_to_one.py

```diff
@@ -196,6 +196,56 @@ def test_send_non_english_message_to_newly_added_contact_on_different_networks(s
         #     self.errors.append("Updated profile picture is not shown in one-to-one chat")
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(111111111111)
+    @marks.high
+    def test_edit_message_in_one_to_one_and_public_chats(self):
+        self.create_drivers(2)
+        device_1, device_2 = SignInView(self.drivers[0]), SignInView(self.drivers[1])
+        device_1_home, device_2_home = device_1.create_user(), device_2.create_user()
+
+        device_2.just_fyi("Create public chat on Device1, send message and edit it then")
+        public_key_1, username_1 = device_1_home.get_public_key_and_username(return_username=True)
+        public_key_2, username_2 = device_2_home.get_public_key_and_username(return_username=True)
+        device_1_home.home_button.click()
+        device_2_home.home_button.click()
+        chat_name = device_1_home.get_random_chat_name()
+        device_1_home.join_public_chat(chat_name)
+        device_1_public_chat = device_1_home.get_chat_view()
+        message_before_edit = "Message BEFORE edit 1"
+        message_after_edit = "Message AFTER edit 2"
+        device_1_public_chat.send_message(message_before_edit)
+        device_1_public_chat.edit_message_in_chat(message_before_edit, message_after_edit)
+        if not device_1_public_chat.element_by_text_part("⌫ Edited").is_element_displayed():
+            self.errors.append('No mark in message bubble about this message was edited')
+
```

Comment:
would be good to add check that you can't edit messages of other users

Reply:
Added.